modules/cms-kit/src/Volo.CmsKit.Public.Web/DefaultHomePageMiddleware.cs

```diff
@@ -12,18 +14,22 @@ public class DefaultHomePageMiddleware : IMiddleware, ITransientDependency
 {
     public async Task InvokeAsync(HttpContext context, RequestDelegate next)
     {
-        if (context.Request.Path.Value == "/")
+        var featureChecker = context.RequestServices.GetRequiredService<IFeatureChecker>();
+
+        if (await featureChecker.IsEnabledAsync(CmsKitFeatures.PageEnable))
```

Comment:
The PR has been closed, but I think it worth to be optimized.
Would it be better to perform a feature check after determining `if (context.Request.Path.Value == "/")`? Since this middleware has a very high priority, it now causes this feature check to be performed on any request, e.g. static file(js,css,...).